README.md

```diff
@@ -4,7 +4,7 @@ Split your [PHPStan baseline](https://phpstan.org/user-guide/baseline) into mult
 
 ```txt
 baselines/
- ├─ loader.neon
+ ├─ _loader.neon
  ├─ empty.notAllowed.neon
  ├─ foreach.nonIterable.neon
  ├─ identical.alwaysFalse.neon
```

Comment:
So this example from the readme would then be realistic regarding the file order.